### DISCUSSION TITLE
organicmaps/tracking module

### DISCUSSION THREAD

**Discussion by cyber-toad** [organicmaps/organicmaps#7329] [Q&A] [answered]
During review of a code I noticed that there is a organicmaps/tracking module that looks like a disabled code to send some track details to a server. It is not enabled for now, is there some intention to keep this code or it could be removed as a cleanup?
Thanks.

**Comment by biodranik** [comment, marked as answer]
This code was used to share real traffic information data with a traffic provider. It may be used in the future with a clear user disclaimer when we integrate online traffic and accident information.

**Comment by PinguDEV-original** [comment]
If you do this, please do it with some OSS traffic provider, that would be awesome :heart:!
